dataCompareR/R/out_generateMismatchData.R

```diff
@@ -18,7 +18,7 @@
 #' Note that this function requires the user to pass in the two data frames used in the inital comparison. If this data
 #' does not match that used for the generation of the dataCompareR object the results produced will not be accurate.
 #' @family dataCompareR.functions
-#' @param x A dataCompareRobject.
+#' @param x A `dataCompareR` object.
```

Comment:
That actually wasn't a typo, but rather the S3 class name (names are difficult). However, we should probably write it as `\code{dataCompareRobject}`

Reply:
Changed.

Reply:
Wrapping it in `\code{}` doesn't make it render nicely on the pkgdown site, but backticks do. :(

Reply:
That's odd since it seems to work for other instances of `\code{}` like in `saveReport` (and since that's the standard way of formatting code in Roxygen)